source/code-snippets/usage-examples/find.js

```diff
@@ -30,6 +30,7 @@ async function run() {
       console.log("No documents found!");
     }
 
+    // replace console.dir with your callback to access individual elements
```

Comment:
thought: I wonder if we could demonstrate a custom callback/lambda here that just calls console.dir so that readers could more easily extrapolate this into their own custom logic. Not necessary, and definitely not optimal or good coding, but it could be worth the tradeoff to demonstrate.

Reply:
Yes, definitely a tough call! I thought about spelling out the callback rather than using this short form, but remembered we had some conversations about what to put here. I think "console.dir" was a recommendation from the drivers engineer team. It might not have been in the context of "put your own callback here", but rather "this idiomatic JS should serve the purpose of printing results". Since the user feedback appears to be related to the misunderstanding of the forEach parameter and not noticing existing information on the page, I think this way might be less prone to additional feedback related to this item.